subiquity/server/controllers/filesystem.py

```diff
@@ -150,27 +150,31 @@ def setup_disk_for_guided(self, target, mode):
         if DeviceAction.TOGGLE_BOOT in DeviceAction.supported(disk):
             self.add_boot_disk(disk)
         if gap is None:
-            return disk, gaps.largest_gap(disk)
+            return gaps.largest_gap(disk)
         else:
             # find what's left of the gap after adding boot
             gap = gaps.within(disk, gap)
             if gap is None:
                 raise Exception('failed to locate gap after adding boot')
-            return disk, gap
+            return gap
```

Comment:
I wonder if this method still has the best name given the change in its signature here?

Reply:
I have a refactor branch that I want to do soon (next?).  I have a slightly better name `setup_gap_for_guided`, but really the refactor branch should sort that out.